js/pages/confirmation.jsx

```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react';
 /* eslint-disable import/first */
```

Comment:
@adborden This was to keep eslint from complaining that we were mixing absolute and relative imports. I figured you probably cared more about the grouping of imports than absolute/relative, but I'm happy to change if I made the wrong call!